src/any_agent/frameworks/any_agent.py

```diff
@@ -217,7 +219,9 @@ async def run_async(
         trace.final_output = final_output
         return trace
 
-    def serve(self, serving_config: A2AServingConfig | None = None) -> None:
+    # Subclass: ServedAnyAgent?
+
+    def serve_a2a(self, serving_config: A2AServingConfig) -> None:
```

Comment:
Please add context to why the default was removed. Docstring is now also out of date.